src/utils/errors.ts

```diff
@@ -372,8 +373,15 @@ export class SubnetAddressError extends AvalancheError {
   }
 }
 
+export class UnknownFormatError extends AvalancheError {
+  constructor(m: string) {
+    super(m, UNKNOWN_FORMAR_ERROR_CODE)
```

Comment:
Fix typo. `UNKNOWN_FORMAR_ERROR_CODE` should be `UNKNOWN_FORMAT_ERROR_CODE`

Reply:
Good spot, fixed.